packages/backend/index.js

```diff
@@ -181,6 +178,10 @@ app.get("/auth-jwt-restricted", jwtAuth, (req, res) => {
   res.send(`all working! 👌. Successfully authenticated request from ${req.address}`);
 });
 
+app.get("/auth-jwt-admin-restricted", jwtAdminAuth, (req, res) => {
```

Comment:
What do you think about changing the middleware names to:
 - `jwtAuth` -> `authOnly`
 - `jwtAdminAuth` -> `adminOnly`

Or something else more descriptive than the current names.

Reply:
I agree!

I've just changed them to `userOnly` and `adminOnly`. https://github.com/moonshotcollective/scaffold-directory/pull/38/commits/6d996b04a91a81fdee6717a4b116b42b4cbf715e

BTW, when I was testing it I got a `returning 401, invalid token. Error: jwt expired` from a jwt generated earlier today. We might want to (tackle #29 || at least increase the TTL (like 1 week/month XD) && (show a message telling the user that he has been logged out and need to re-sign)